packages/@aws-cdk/aws-lambda/lib/layers.ts

```diff
@@ -54,15 +54,17 @@ export interface ILayerVersion extends IResource {
   export(): LayerVersionAttributes;
 
   /**
-   * Grants usage of this layer to specific entities. Usage within the same account where the layer is defined is always
-   * allowed and does not require calling this method. Note that the principal that creates the Lambda function using
-   * the layer (for example, a CloudFormation changeset execution role) also needs to have the
-   * ``lambda:GetLayerVersion`` permission on the layer version.
+   * Add permission for this layer version to specific entities. Usage within
+   * the same account where the layer is defined is always allowed and does not
+   * require calling this method. Note that the principal that creates the
+   * Lambda function using the layer (for example, a CloudFormation changeset
+   * execution role) also needs to have the ``lambda:GetLayerVersion``
+   * permission on the layer version.
    *
    * @param id the ID of the grant in the construct tree.
    * @param grantee the identification of the grantee.
    */
-  grantUsage(id: string, grantee: LayerVersionUsageGrantee): ILayerVersion
+  addPermission(id: string, grantee: LayerVersionPermission): void;
```

Comment:
Parameter name and type name no longer agree.

Reply:
renamed to "permission"